ksql-rest-app/src/test/java/io/confluent/ksql/rest/client/KsqlRestClientTest.java

```diff
@@ -200,6 +200,33 @@ public void shouldThrowOnInvalidServerAddress() {
     new KsqlRestClient("not-valid-address");
   }
 
+  @Test
+  public void shouldParseSingleServerAddress() {
+    final String singleServerAddress = "http://singleServer:8088";
+    URI singleServerUri = null;
+    try {
+      singleServerUri = new URI(singleServerAddress);
+    } catch (Exception ex) {}
+    try (KsqlRestClient client = new KsqlRestClient(singleServerAddress)) {
+      assertThat(client.getServerAddress(), is(notNullValue()));
+      Assert.assertEquals(singleServerUri, client.getServerAddress());
+    }
+  }
+
+  @Test
+  public void shouldParseMultipleServerAddresses() {
+    final String firstServerAddress = "http://firstServer:8088";
+    final String multipleServerAddresses = firstServerAddress + ",http://secondServer:8088";
+    URI firstServerUri = null;
+    try {
+      firstServerUri = new URI(firstServerAddress);
+    } catch (Exception ex) {}
+    try (KsqlRestClient client = new KsqlRestClient(multipleServerAddresses)) {
+      assertThat(client.getServerAddress(), is(notNullValue()));
+      Assert.assertEquals(firstServerUri, client.getServerAddress());
+    }
+  }
```

Comment:
Would be good to add a test that covers what happens if one of the later addresses fails to parse.

Reply:
Done! Changed the existing test cases and added a new one to cover the scenario where one of the endpoints provided is invalid.